ai21/clients/studio/resources/studio_chat.py

```diff
@@ -24,6 +26,11 @@ def create(
         count_penalty: Optional[Penalty] = None,
         **kwargs,
     ) -> ChatResponse:
+        warnings.warn(
+            "This method is deprecated. Please use the `chat.completions.create` method in the client instead.",
```

Comment:
Let's put a pin on that. Deprecation might not happen on the first step

Reply:
You mean open a ticket so we won't forget to deprecate? or to wait with the deprecation warning until we know a more specific deprecation date?

Reply:
The latter